themes/default/content/docs/languages-sdks/python/python-blocking-async.md

```diff
@@ -0,0 +1,199 @@
+---
+title_tag: "Blocking and Async Python with Pulumi | Languages & SDKs"
```

Comment:
Could we add a note to the Python page linking to this?

Reply:
Done